src/MicrosoftTeams.ts

```diff
@@ -861,6 +861,24 @@ export function showNotification(
   sendMessageRequest(parentWindow, "showNotification", params);
 }
 
+/**
+ * @private
+ * Hide from docs.
+ * ------
+ * execute deep link API.
+ * @param deepLink deep link.
+ */
+export function executeDeepLink(deepLink: string): void {
+  ensureInitialized(frameContexts.content);
+  const params = [deepLink];
```

Comment:
NIT: You could just put this inline below rather than using an intermediate `params` variable.